docs/source/developer_guide/contribution/index.md

```diff
@@ -58,6 +58,17 @@ cat requirements.txt | grep -Ev '^#|^--|^$|^-r' | while read PACKAGE; do pip ins
 bash format.sh ci
 ```
 
+Note: If you did not create the Python .venv in the "Running lint Locally" section above,
```

Comment:
It's quite strange. Even users run lint without .venv, mypy should find the requirement in system PYTHONPATH, why users need to add the pwd to PYTHONPATH?